templates/statefulset.yaml

```
@@ -186,6 +185,8 @@ spec:
 
           echo NiFi running with PID ${nifi_pid}.
           wait ${nifi_pid}
+          /opt/nifi/nifi-toolkit-current/bin/tls-toolkit.sh standalone -n '{{.Values.properties.release}}-nifi-0.{{.Values.properties.release}}-nifi-headless.{{.Values.properties.namespace}}.svc.cluster.local' -C '{{.Values.auth.ldap.admin}}' -o '/opt/nifi/nifi-current/conf/' -P truststorePasswdfadi  -S keystorePasswdfadi  --nifiPropertiesFile /opt/nifi/nifi-current/conf/nifi.properties
```

```suggestion
${NIFI_TOOLKIT_HOME}/bin/tls-toolkit.sh standalone -n "${FQDN}" -C '{{.Values.auth.ldap.admin}}' -o "${NIFI_HOME}/conf/" -P truststorePasswdfadi  -S keystorePasswdfadi  --nifiPropertiesFile "${NIFI_HOME}/conf/nifi.properties"
```

Plus templating the passwords would be good.

---

thank you, but I tested it and it doesn't work, i get the error << no valid keystore >>